.github/workflows/ci.yaml

```diff
@@ -48,24 +50,19 @@ jobs:
       - name: Create cache file
         run: echo '${{ matrix.python-version }}' > ./matrix-file.txt
 
-      - uses: pdm-project/setup-pdm@v4
-        name: Set up PDM
-        with:
-          python-version: ${{ matrix.python-version }}
-          allow-python-prereleases: false
-          cache: true
-          cache-dependency-path: |
-            ./pdm.lock
-            ./matrix-file.txt
+      - name: Install uv
+        uses: astral-sh/setup-uv@v4
+        version: "0.5.10"
```

Comment:
Is this something that needs to be manually incremented for version updates or does dependabot pick this up?

Reply:
I think that it will have to be increment manually, because it is not included in the list of dependencies

I looked at the [workflow in the litestar](https://github.com/litestar-org/litestar/blob/main/.github/workflows/ci.yml), the version was also fixed there

Reply:
Ah, in that case.  Let's leave it as is then.  Thanks.

Reply:
Take a look at the output from this [run](https://github.com/litestar-org/litestar-fullstack/actions/runs/12417654825) I just triggered.  I'll go dig into the Litestar CI now as well.

Reply:
found a mistake, i missed the `with` block